pkg/bundle/sync.go

```diff
@@ -307,6 +308,7 @@ type pkcs12Encoder struct {
 }
 
 func (e pkcs12Encoder) encode(trustBundle string) ([]byte, error) {
+	fmt.Printf("trusted bundle %s", trustBundle)
```

Comment:
Ah, yes, sorry for that. 
By the way, how do you debug operator? The only one way I found it is just put changes and re-deploy operator by run `make ensure-trust-manager` command.

Reply:
I have GoLand (by JetBrains) - with an integrated debugger. 😁

Reply:
Yes, for standalone binaries it's possible to use either delve or even dgb with IDE, however for container workloads need to use something like `CMD ["/dlv", "--listen=:40000", "truster-manager-operator-binary"]` then try to connect from

Reply:
But why do you want to debug the _container workload_ for this? 🤔

Reply:
As far as I understand `make ensure-trust-manager` deploy trust-manager operator in k8s cluster, so that's why at least `kind` should be present on he local machine. In my case trust-manager is deployed in the `kind k8s cluster`, therefore Deployments and Pod resource are created. That's what I see.

Reply:
I have feelings that trust-manager which is running locally can leverage k8s API by utilizing local kubeconfig :)

Reply:
I am only using https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/envtest locally. The only time I need to consider remote debugging is to debug a running application in one of our real clusters.

Reply:
Understand, however I'm almost new in this project so for me important to trace request from the very beginning to the end, then I', better understanding how things work.